dev-python/pip/pip-19.3.1.recipe

```diff
@@ -20,8 +20,8 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
-PYTHON_PACKAGES=(python python3 python38 python39)
-PYTHON_VERSIONS=(2.7 3.7 3.8 3.9)
+PYTHON_PACKAGES=(python python3 python38 python39 python310)
```

Comment:
Can we please use `python3.10` for python 3.10, at least, even if we can't for the other versions?